propolis-cli/src/main.rs

```diff
@@ -120,6 +120,8 @@ fn main() {
         let hdl = machine.get_hdl();
         let chipset = hw::chipset::i440fx::I440Fx::create(Arc::clone(&hdl));
         chipset.attach(mctx);
+        let chipset_id = inv.register_root(chipset.clone(), "chipset".to_string())
```

Comment:
Changes in this file should be limited to the conversion to the new interface API